osal/win32/osal_win32.h

```diff
@@ -29,6 +29,7 @@
     }                                                                         \
   } while (0)
 
+struct timezone;
 int osal_gettimeofday (struct timeval *tv, struct timezone *tz);
```

Comment:
The declaration of osal_gettimeofday can be removed hence no need to add forward declaration of struct timezone.
I believe osal_win32.h might be removed all together